zerver/tests/test_email_notifications.py

```diff
@@ -1338,15 +1338,15 @@ def test_sender_name_in_missed_message(self) -> None:
         self.assertIn("Iago:\n> @**King Hamlet**\n\n--\nYou are", mail.outbox[0].body)
         # If message content starts with <p> tag the sender name is appended inside the <p> tag.
         self.assertIn(
-            '<p><b>Iago</b>: <span class="user-mention"', mail.outbox[0].alternatives[0][0]
+            '<p style="font-family: sans-serif;font-size: 14px;font-weight: normal;margin: 10px 0;"><b>Iago</b>: <span class="user-mention"', mail.outbox[0].alternatives[0][0]
```

Comment:
Since it's just an `assertIn`, let's just pull the `p` tag out of the assertion, so we don't have all the CSS in this test.

(This is probably a good option for most of these).

I think we want at a couple tests that include all the interpolated CSS in it in this file, to verify that we are doing the CSS interpolation, but generally it's just going to make a test harder to read and we should just check for `><b>Iago</b>: <span class="user-mention"` to see that this is working as intended? Hmm, that might not work for this test case in particular, because the assertion suggested by the comment IS about the `p` page, but maybe you can apply this idea to the other ones with an image tag or something.

(Or maybe we should just replace this assertion with an HTML assertion using `xpath` or something)